docker-compose.yml

```diff
@@ -33,6 +33,7 @@ web:
     LORE_ADMIN_EMAIL:
     LORE_CAS_URL:
     CELERY_ALWAYS_EAGER: False
+    CELERY_RESULT_BACKEND: redis://redis:6379/4
```

Comment:
I'm not sure how long these results last. @carsongee ?

This was added to allow our docker containers to keep track of Celery tasks so that the front end can watch their progress.